flight/targets/flyingf3/fw/pios_board.c

```diff
@@ -710,6 +710,16 @@ void PIOS_Board_Init(void) {
 	if (PIOS_BMP085_Test() != 0)
 		panic(5);
 #endif /* PIOS_INCLUDE_BMP085 && PIOS_INCLUDE_I2C */
+#if defined(PIOS_INCLUDE_ADC)
```

Comment:
This change overall looks good to me.  But for these lines I suggest DONT_BUILD_IF instead of PIOS_Assert (compile time assert instead of runtime hang).  It'll have to be outside the function body though.